uilib-docs/package.json

```diff
@@ -6,7 +6,7 @@
   "author": "Kyle Mathews <mathews.kyle@gmail.com>",
   "dependencies": {
     "classnames": "^2.2.6",
-    "gatsby": "2.23.22-static-query-template.8",
```

Comment:
Did you test this on production?
We have this open bug: https://github.com/gatsbyjs/gatsby/issues/26038
(They want us to validate something but I haven't gotten around to it...)

Reply:
It seems to work now. 
The current docs were generated from this branch.

Reply:
I'm not sure it has been, locally I had `Button` and I don't see it in production.

Reply:
I believe someone had deployed the docs from master and "broke" it :/ 
I just triggered docs again from this branch and now you can see the Button